### ISSUE TITLE
Federation menu icons not showing for profile picture, full name and email.

### ISSUE THREAD

**Issue by korn806** [nextcloud/server#21071]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Expected behaviour
Federation menu icons should show next to every field of personal info

### Actual behaviour
Federation menu icons not showing for profile picture, full name and email.

### Server configuration

**Operating system:**
Ubuntu 20.04

**Web server:**
Apache 2.4.43

**Database:**
MariaDB 10.3.22

**PHP version:**
7.3.18

**Nextcloud version:** (see Nextcloud admin page)
18.0.4

**Updated from an older Nextcloud/ownCloud or fresh install:**
update

**Where did you install Nextcloud from:**
nextcloud.com

![ss3_nextcloud](https://user-images.githubusercontent.com/29353104/82692597-4d8b7500-9c60-11ea-88ca-89e92f98ae23.png)

**Comment by fzirker** [comment]
I have encountered the same problem. 

Apparently, the problem only appears with already existing user accounts. I created a new user account for testing. There, I can apply the privacy setting (federation menu) to all attributes. 

I only noticed the error because if the full name is not set to public, no username is displayed in a File drop (upload only) share.

**Comment by fzirker** [comment]
Further investigations: 

I looked at the HTML and found an div-element whith the class "federation-menu". The inner span for the icon had the CSS class "hidden". I removed that class and then the icon apears. I was able to set my desired privacy setting. This is also taken over correctly when reloading the page. 

Eventually there is something wrong in the js file wich loads the stored settings. iI found that n the apps/settings/js/federationscopemenu.js the icons are set with the CSS class "hidden" at the beginning. Apparently the "hidden" class is not removed correctly when no value is found.
